src/components/label/ohri-label.component.tsx

```diff
@@ -1,17 +1,24 @@
 import React from 'react';
 import { DefinitionTooltip } from '@carbon/react';
 import styles from './ohri-label.scss';
+import { useTranslation } from 'react-i18next';
+import i18next from '../translator/i18n';
 
 type LabelProps = {
   value: string;
   tooltipText?: string;
 };
 
 export const OHRILabel: React.FC<LabelProps> = ({ value, tooltipText }) => {
+  const { t, i18n } = useTranslation('ohri', { i18n: i18next });
```

Comment:
Am not sure if the key `ohri` was meant to imply something specific but i believe we should opt for more generic names and not organization specific naming.

Reply:
There is an existing i18n instance, I was not able to override to reference the forms. 
So, within it does not lookUp forms resources that has subforms with locale. 
We can see together how to override that.